### DISCUSSION TITLE
How to support the application of GAP potentials under GPU acceleration now

### DISCUSSION THREAD

**Discussion by 1227658436** [libAtoms/QUIP#638] [Q&A] [answered]
I previously installed LAMMPS with the QUIP package, but during runtime, I noticed that the GPU utilization remained consistently at 0%. Hence, I have a question for everyone: How can I address this issue and enable GPU acceleration for the application of GAP potentials

**Comment by gabor1** [comment, marked as answer]
GAP is not written for GPU. It is unlikely that we will port it. There is an effort to port TurboGAP to GPU, but I don't know how far along it is. I recommend that you have a look at our [https://github.com/ACESuit/MACE](https://github.com/ACESuit/MACE) potential, that is specifically written to run on GPUs and is in many ways represents an advance on GAP (in terms of accuracy, transferability and speed. GAP fits are faster if you have thousands of CPUs)
